YMCacheTests/YMMemoryCacheSpec.m

```diff
@@ -4,6 +4,14 @@
 
 #import <YMCache/YMCache.h>
 
+void pumpRunLoop(NSTimeInterval forSeconds) {
+    NSDate *stopDate = [NSDate dateWithTimeIntervalSinceNow:forSeconds];
+    NSTimeInterval stopInterval = stopDate.timeIntervalSinceReferenceDate;
+    while (stopInterval >= [NSDate date].timeIntervalSinceReferenceDate) {
+        [[NSRunLoop currentRunLoop] runUntilDate:stopDate];
+    }
+}
+
 SpecBegin(YMMemoryCacheSpec)
 
 fdescribe(@"YMMemoryCache", ^{
```

Comment:
I blame Specta for choosing a character with aggressive kerning for it's focus key.